app/controllers/school/devices_controller.rb

```diff
@@ -4,8 +4,11 @@ def order
     # branch of the user journey
     # if false && @school.can_order_devices?
     # else
```

Comment:
i assume we can now delete this commented code as you've implemented it?

Reply:
Good spot